ci/lib/stage-build-gsc.jenkinsfile

```diff
@@ -222,6 +222,26 @@ stage('Build and Test Workloads') {
                 docker rmi verifier:latest gsc-ovms-image &>/dev/null
             '''
         }
+
+
+        try {
+            sh '''
+                echo 'Executing Build Openvino Example'
```

Comment:
The print statement is incorrect, Refer above examples. It should be aligned otherwise nightly parser won't be able to parse the results